packages/logger/src/index.ts

```diff
@@ -4,16 +4,21 @@ import { IncomingMessage as Request, ServerResponse as Response, METHODS } from
 
 export type LoggerProperties = Partial<{
   methods: string[]
+  outputConfiguration: {
```

Comment:
Every property in the config must be optional, here `color` and `output` seem to be both required.

EIther put `?` in the end of properties names or use `DeepPartial` ([ref](https://stackoverflow.com/a/49936686))

Reply:
Wouldn't this mean changing `LoggerProperties` to an interface? since there is no meaning when using `Partial` combined with `?`

Reply:
@Arnovsky hmm then probably u're right, it makes more sense to change it to interface.

Also, please cover ur new functionality in tests (if there are none yet) :)